assets/css/main.scss

```diff
@@ -505,6 +505,17 @@ body.layout-docs {
   .doc-main {
     flex: 1 1 auto;
   }
+
+  header {
+    position: static;
```

Comment:
Wew that was fast! was about to push another commit to remove this unnecessary line :)